hafta1/hafta1.md

```diff
@@ -609,11 +609,29 @@ Bu yüzden lütfen dışarıdan bir uygulama kuracaksınız, forumlara ya da you
 
 ---
 
+## AUR(Arch User Repository) Nedir ?
+
+AUR, archlinux topluluğunun geliştirdikleri uygulama paketlerini, kütüphaneleri ve daha bir çok şeyi tüm archlinux topluluğuyla paylaştığı repodur. AUR da bulunan paketler ana repoda yoktur. AUR da bulunan paketler topluluk tarafından oylanarak olumlu yönde oy almış paketler ana repoya taşınabilir.
+
+## AUR Helpers
+
+Ana repodan bir paket indirmek istediğimizde komut satırına pacman yazıyoruz. Peki Arch User reposundan bir paket indirmek istediğimizde ne kullanmamız gerekiyor ?
+Burada devreye AUR helpers giriyor. AUR içinde bir paketi aramak yada indirmek için helpersları kullanırız. 
```

Comment:
AUR'dan elle paket kurdurdaktan sonra AUR Helper için başlık açıp önce ne olduğunu ve AUR Helper uygulamalarınını yazdır sonra.

Reply:
İndirmek değil yüklemek ifadesini kullanalım. İndirmek anlamı başka bir kelime.